aten/src/ATen/native/mkldnn/xpu/detail/oneDNNContext.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <ATen/Config.h>
+
+#include <c10/core/Device.h>
+#include <c10/xpu/XPUFunctions.h>
+#include <c10/xpu/XPUStream.h>
+
+#include <oneapi/dnnl/dnnl.hpp>
+#include <oneapi/dnnl/dnnl_sycl.hpp>
+#include <vector>
+
+namespace at::native::onednn {
+
+// Keep non-static and non-inline
+bool set_onednn_verbose(int level);
+
+static inline dnnl::memory make_onednn_memory(
```

Comment:
we need these APIs to be public, if considering to reuse them in IPEX, right?

Reply:
We need to add TORCH_API for it?

Reply:
Sorry that I may need further explanation on this. 
1. If the problem is the `static inline` declaration, I think the user could directly `#include <detail/oneDNN>` or `#include <detail/oneDNNContext.h>` like our op in aten level?
2. If the concern lies at linkage, so we need to put it into a `cpp` source file and declared as `TOCH_API`?

Reply:
Why we do not move all impl in this file to corresponding ccp file, then, expose the APIs by TORCH_API?

Reply:
A commit has been pushed, 
1. Runtime related API is exposed, like `xpu_onednn_memory`, `GpuEngineManger`, `GpuStreamManager`
2. Function in `Utils.h` is mostly a part of onednn integration specific code, users could easily rewrite in their repos. I suggest we do not export as an API of TORCH.